### PR TITLE
Added renewed models and sample_data task, completed Photogram-industrial-2

### PR DESCRIPTION
**Pull request by brvarner**

### CODE REVIEW

**Comment by brvarner** [comment]
> _ No description provided. _

I've added Adolfo to my repo, so he'll hopefully provide peer review

**Review by AdolfoNava — APPROVED** [review]
Looks good to me

File: .vscode/extensions.json
Comment (thierrychau) [inline .vscode/extensions.json:1]: This probably should not be committed to your repo.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```
Comment (thierrychau) [inline app/controllers/application_controller.rb:2]: Nitpick: that was not required in these assignments. All good though.

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :own_photos, class_name: "Photo", foreign_key: "owner_id"
+  has_many :comments, foreign_key: "author_id"
+  has_many :likes, foreign_key: :fan_id
+
+  has_many :sent_follow_requests, foreign_key: :sender_id, class_name: "FollowRequest"
```
Comment (thierrychau) [inline app/models/user.rb]: Might need to add `dependent: :destroy` to take care of parentless records.

File: app/views/layouts/application.html.erb
Comment (thierrychau) [inline app/views/layouts/application.html.erb:1]: I don't think these changes were part of this pull request. Try to keep PR focused on a single object to reduce noise, it makes review easier. This PR already has a lot of files :). It's all good for this time though, but keep it in mind in the future.

File: app/views/shared/_cdn_assets.html.erb
Comment (thierrychau) [inline app/views/shared/_cdn_assets.html.erb:1]: Noise!

File: app/views/shared/_navbar.html.erb
Comment (thierrychau) [inline app/views/shared/_navbar.html.erb:1]: Noise!

File: config/initializers/simple_form.rb
Comment (thierrychau) [inline config/initializers/simple_form.rb:1]: Do we need that?

File: config/locales/simple_form.en.yml
Comment (thierrychau) [inline config/locales/simple_form.en.yml:1]: Do we need that?

File: lib/templates/erb/scaffold/_form.html.erb
Comment (thierrychau) [inline lib/templates/erb/scaffold/_form.html.erb:1]: what's this?

**Review by thierrychau — CHANGES_REQUESTED** [review]

File: app/views/shared/_cdn_assets.html.erb
Comment (brvarner) [reply app/views/shared/_cdn_assets.html.erb:1]: I need it!

File: lib/templates/erb/scaffold/_form.html.erb
Comment (brvarner) [reply lib/templates/erb/scaffold/_form.html.erb:1]: not sure tbh, I tried devise:views at some point so I'm wondering if that created some of these more obscure scaffold files. I ran the devise:delete command to get rid of the views but apparently it didn't get everything 🤦🏾I'm deleting this and the other files from my repo

**Review by thierrychau — APPROVED** [review]
Good to merge

**Pull request merged**